content/news/XXXX-XX-XX-mixxx-2-3-1-released.md

```diff
@@ -0,0 +1,19 @@
+title: Mixxx 2.3.1 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.1, release announcement
+status: draft
+comments: yes
+
+We are happy to announce first bugfix release of [Mixxx 2.3.1](https://github.com/mixxxdj/mixxx/releases/tag/2.3.1)!
```

Comment:
```suggestion
We are happy to announce the release of [Mixxx 2.3.1](https://github.com/mixxxdj/mixxx/releases/tag/2.3.1)!
```

Reply:
"first release of Mixxx 2.3.1" is odd; it implies there will be other releases of Mixxx 2.3.1.